### DISCUSSION TITLE
Speed up transcoding?

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#150] [Q&A] [answered]
How can I speed up transcoding?

I'm currently using the standard docker deployment on a single 4vCPU 16GB RAM LightSail server. For 1080p video, transcoding can take up to 2 times the video duration. The server will hit 100% CPU utilization in these cases.

But I'm hoping to load up a few TB of media at the kick-off.
How can I speed up transcoding?

**Comment by mgogoulos** [comment, marked as answer]
It's true that some profiles take time, I've noticed same time more or less for 1080.

You can edit these profiles, if you are familiar with ffmpeg but doubt that there will be any noticeable difference for high profiles as 1080 etc. For example you can change the speed presets and put the fastest but then this creates not good quality (while the size will be high)

Another solution would be to disable all profiles (let only 240 and 480), then bulk upload content, so it all appears very soon, since the 240+480 don't take much time to encode. 
Then enable all other profiles (as 1080) and run through Django shell 3-4 lines of code to let the system know that new profiles are added and that the existing videos need transcoded to these profiles too.

There's also remote encoders capabilities but this is not part of the open source version.
